source/gui/configProfiles.py

```diff
@@ -171,11 +171,11 @@ def onDelete(self, evt):
 		index = self.profileList.Selection
 		if gui.messageBox(
 			# Translators: The confirmation prompt displayed when the user requests to delete a configuration profile.
-			_("Are you sure you want to delete this profile? This cannot be undone."),
+			_("This profile will be permanently deleted, this action cannot be undone."),

```

Comment:
Please split this into two sentences; i.e. "This profile will be permanently deleted. This action cannot be undone."